src/features/status-bar/StatusBarBase.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import { FC, ReactNode } from 'react';
+import { IncidentImpact } from 'statuspage.io';
+import { Icon } from '@/ui/Icon';
+import { BsExclamationCircle, BsExclamationTriangle } from 'react-icons/bs';
+import { getColor } from '@/features/status-bar/utils';
+import { Box } from '@/ui/Box';
+import { Flex } from '@/ui/Flex';
+
+export const StatusBarBase: FC<{ impact: IncidentImpact; content: ReactNode }> = ({
+  content,
+  impact,
+}) => {
+  if (impact === IncidentImpact.None) return <h1>something</h1>;
```

Comment:
Is this `<h1>something</h1>` intentional? It seems maybe like a placeholder that didn't get updated

Reply:
That's my bad, thank you for caching this!